packages/common.vm/tools/chocolateyinstall.ps1

```diff
@@ -106,31 +106,6 @@ if (-Not (Test-Path $toolListDir)) {
     Write-Host -ForegroundColor Green "[+] Created folder:" $toolListDir
 }
 
-
-# ################################################################################################ #
-# Setup the default tool list directory shortcut and env var if it doesn't exist
-# ################################################################################################ #
-$envVarName = "TOOL_LIST_SHORTCUT"
```

Comment:
With the removal of this, please make sure to update the other references to it and the common.vm `readme.md` page for documentation. Namely:

- [ ] https://github.com/mandiant/VM-Packages/blob/4f8d53431aa112f3c32055c23c163eae4ec2e89c/packages/common.vm/README.md?plain=1#L21-L25
- [ ] https://github.com/mandiant/VM-Packages/blob/4f8d53431aa112f3c32055c23c163eae4ec2e89c/packages/common.vm/tools/chocolateyuninstall.ps1#L20
- [ ] https://github.com/mandiant/VM-Packages/blob/4f8d53431aa112f3c32055c23c163eae4ec2e89c/packages/common.vm/tools/vm.common/vm.common.psm1#L882